man/print.data.table.Rd

```diff
@@ -43,6 +45,7 @@
   \item{col.names}{ One of three flavours for controlling the display of column names in output. \code{"auto"} includes column names above the data, as well as below the table if \code{nrow(x) > 20}. \code{"top"} excludes this lower register when applicable, and \code{"none"} suppresses column names altogether (as well as column classes if \code{class = TRUE}. }
   \item{print.keys}{ If \code{TRUE}, any \code{\link{key}} and/or \code{\link[=indices]{index}} currently assigned to \code{x} will be printed prior to the preview of the data. }
   \item{trunc.cols}{ If \code{TRUE}, only the columns that can be printed in the console without wrapping the columns to new lines will be printed (similar to \code{tibbles}). }
+  \item{trunc.char}{If a number is provided as the value for \code{"datatable.prettyprint.char"}, it sets a limit for the length of strings processed by \code{char.trunc}. Strings longer than this limit have their excess characters removed, and "..." is appended to indicate omitted content.}
```

Comment:
```suggestion
  \item{trunc.char}{ If \code{"datatable.prettyprint.char"} is a number greater than 0, \code{data.table.print} limits the lengths of strings to be equal to or less than \code{"datatable.prettyprint.char"}. Strings longer than this limit have their excess characters removed, and "..." is appended to the end of the string to indicate omitted content. Strings shorter than this limit are left unchanged. }
```

Reply:
Not sure if there are any rules regarding this but my suggested description is quite long - could omit some parts of it if we want to keep it simple. @MichaelChirico any thoughts?

Something like:

```R
\item{trunc.char}{ If \code{"datatable.prettyprint.char"} is a number greater than 0, \code{data.table.print} limits the lengths of printed strings to be equal to or less than \code{"datatable.prettyprint.char"} by removing excess characters and concatenating "..." to the end.
```